oxidation/client/.eslintrc.js

```diff
@@ -17,8 +17,8 @@ module.exports = {
     'cypress'
   ],
   rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-console': 'off',
+    'no-debugger': 'error',
```

Comment:
Would be noisy during debug session, Why not keeping the system where a env variable can disable this rule ?